requirements.txt

```diff
@@ -0,0 +1,2 @@
+cmake>3.1
+numpy>1.16
```

Comment:
What's wrong with versions 1.14 & 1.15 of numpy?

Reply:
good point, those work too